Neurocorrelates causes MABE build to fail on Mac

When building MABE with neurocorrelates on MacOS, the following error is thrown:

`./Analyze/neurocorrelates.cpp:11:7: error: no member named 'log2' in namespace 'std'; did you mean '::log2'?`

The issue seems to be `log2` and `pow` are not valid members of the `std` namespace. Since `include <math.h>` is declared, deleting the following line fixes the issue with no further errors:

`11: using std::log2; using std::pow;`

--- a/Analyze/neurocorrelates.cpp
+++ b/Analyze/neurocorrelates.cpp
@@ -7,7 +7,8 @@ using std::vector;
 using std::map; using std::pair; 
 #include <algorithm> 
 using std::copy; using std::min; using std::max; 
-#include <math.h>
+#include <cmath>
+using std::log2; using std::pow;
 #include <iostream>
 using std::cout; using std::endl;
 

--- a/Analyze/neurocorrelates.cpp
+++ b/Analyze/neurocorrelates.cpp
@@ -8,7 +8,6 @@ using std::map; using std::pair;
 #include <algorithm> 
 using std::copy; using std::min; using std::max; 
 #include <math.h>
-using std::log2; using std::pow; 
 #include <iostream>
 using std::cout; using std::endl;
 

